assets/js/base/components/shipping-rates-control/style.scss

```diff
@@ -27,9 +27,8 @@
 	color: $core-grey-dark-400;
 }
 
-.wc-block-shipping-rates-control__no-results {
-	margin-bottom: 0;
-	font-style: italic;
+.wc-block-shipping-rates-control__no-results-notice {
+	margin-bottom: $gap !important;
```

Comment:
Here again, I think we could remove `!important` adding an extra class to the selector.

Reply:
@mikejolley this was marked as solved but I think those lines didn't change, right? Was that an overlook?